datasets/mx/tax_offender/mx_tax_offender.yaml

```diff
@@ -0,0 +1,43 @@
+title: Mexico Tax Offenders
+entry_point: crawler.py
+prefix: mx-taxo
+coverage:
+  frequency: weekly
+  start: 2024-09-05
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Entities that are considered tax offenders according to Article 69 of the Federal Tax Code
+description: |
+  The Tax Administration Service publishs information about individuals and entities identified
+  as tax offenders under Article 69 of the Federal Tax Code in Mexico. This dataset includes
+  details such as names, tax identification numbers, and the nature of the offenses.
+
+publisher:
+  name: Servicio de Administración Tributaria
+  acronym: SAT
+  description: Tax Administration Service
+  url: https://www.sat.gob.mx/home
+  official: true
+  country: "mx"
+url: "http://omawww.sat.gob.mx/cifras_sat/Paginas/datos/vinculo.html?page=ListCompleta69.html"
+data:
+  url: http://omawww.sat.gob.mx/cifras_sat/Paginas/datos/vinculo.html?page=ListCompleta69.html
+  format: HTML
```

Comment:
When I try to export to check the number of entities I get the following error, do you know what it could be?

<img width="973" alt="Screenshot 2024-09-12 at 11 27 51" src="https://github.com/user-attachments/assets/ea826aae-3d9d-4e38-ac5b-b9c2f43a0af3">

Reply:
this happens if you generate the same ID for an entity with two different incompatible schemata, e.g. Company and Person.
I was wondering about using Company in any case. Some of the entities might just be legal entities and not companies per se.
But this might also indicate a bug, so I think before changing the Company schema, see if we're correctly generating the same ID for two different rows of data.

Reply:
It worked now, it was because it was choosing the wrong schema for a person that appears twice, but I fixed it in the commit https://github.com/opensanctions/opensanctions/pull/1226/commits/964080ea17b5a71fc71277c9d6ccffadebfe045d